core/src/main/java/org/infinispan/commands/read/EntrySetCommand.java

```diff
@@ -57,6 +57,10 @@ public LoadType loadType() {
 
    @Override
    public Set<CacheEntry<K, V>> perform(InvocationContext ctx) throws Throwable {
+      Object lockOwner = ctx.getLockOwner();
+      if (ctx.getLockOwner() != null) {
+         return new BackingEntrySet<>(cache.getAdvancedCache().lockAs(lockOwner));
```

Comment:
Yes, iirc the locks don't count invocations, sadly. This was something I had complained about before. To be honest though I don't think it is worth it to change how that behaves here. I can just throw an exception if `lockOwner` was explicitly set before invoking `lockedStream` WDYT?